common/autoware_ad_api_msgs/routing/srv/SetRoute.srv

```diff
@@ -0,0 +1,7 @@
+std_msgs/Header header
+geometry_msgs/Pose[<=1] start
```

Comment:
By the way, do we really need `start`? Could you show me the use case?

Reply:
To set the route before localization is complete or during manual driving.

Reply:
@mitsudome-r @yukkysaito Do you think these use cases are necessary? Then, it's okay.

Reply:
umm...
It may allow for unintended uses. for example, the ego car is far from the route.


If the start pose is needed, it may cause some trouble. 
In places where routes are diverged, it's hard to know which the ego is. It depend on the nearest search algorithm.

Reply:
Discussion result with @isamu-takagi @mitsudome-r @yukkysaito 
- Autoware.Auto has used `start` to adjust the position for a specific algorithm, which is not good.
- We can remove `start` for Autoware Core/Universe.

Reply:
Remove start pose https://github.com/autowarefoundation/autoware.universe/pull/1494/commits/e76935a157cbef28762b9e5aacb85c3e87ad3e71. 
It remains in Route.msg as the self pose at calculation.